app.css

```diff
@@ -0,0 +1,131 @@
+html {
+  box-sizing: border-box;
+}
+
+*, *::before, *::after {
+  box-sizing: inherit;
+
+  /*border: red 2px solid !important;*/
+  /*padding: 4px !important;*/
+}
+
+body {
+  margin: 0;
+  font-family: "Source Sans Pro", sans-serif;
+  font-weight: 200;
+}
+
+ul {
+  padding-left: 0;
+  list-style: none;
+}
+
+h1, h2, h3, h4 {
+  font-weight: 300;
+}
+
+img {
+  max-width: 100%;
+}
+
+a {
+  text-decoration: none;
+}
+
+.nav-bar {
+  background: #2aa3a3;
+  padding-top: 2rem;
+  margin: auto;
+  width: 100%;
+  overflow: auto;
+}
+
+.nav-group {
+  font-size: 1.2rem;
+  margin: 1rem 6rem;
+  width: 13%;
+  float: left;
+  overflow: auto;
+}
+
+.nav-links {
+  margin: 1rem 6rem;
+  color: inherit;
+  width: 13%;
+  float: right;
+}
+
+
+ul.nav-group {
+  margin: 1rem -0.5rem;
+}
+
+.nav-group__item {
+  margin: 0 0.5rem;
+  float: right;
+  opacity: 0.6;
+}
+
+.nav-group__item a {
+  color: black;
+}
+
+.nav-group__item:hover {
+  opacity: 1;
+}
+.row {
+  margin: 5rem;
+  overflow: auto;
+}
+.row__column-1-3 {
+  margin: 1rem;
+width: calc( 100% / 3 - 1rem - 1rem );
+}
+.row__column-2-3 {
+  margin: 1rem;
+  width: calc( 100% / 3 - 1rem - 1rem );
+}
+
+.profile {
+  float: left;
+  border: .05em black solid;
+  border-radius: 5%;
```

Comment:
Use an `rem` value otherwise the border corners will be elliptical instead of rounded circles.